programs/vote_api/src/vote_state.rs

```diff
@@ -380,25 +380,6 @@ pub fn create_account(
     vote_account
 }
 
-// utility function, used by solana-genesis, tests
-pub fn create_bootstrap_leader_account(
-    vote_pubkey: &Pubkey,
-    node_pubkey: &Pubkey,
-    commission: u8,
-    vote_lamports: u64,
-) -> (Account, VoteState) {
-    // Construct a vote account for the bootstrap_leader such that the leader_scheduler
-    // will be forced to select it as the leader for height 0
-    let mut vote_account = create_account(&vote_pubkey, &node_pubkey, commission, vote_lamports);
-
-    let mut vote_state: VoteState = vote_account.state().unwrap();
-    // TODO: get a hash for slot 0?
-    vote_state.process_slot_vote_unchecked(0);
```

Comment:
this line was added in this commit c5a74ada05a6a8b602841792303c2a82dd762f53, dunno why, removing now ;)

Reply:
🎁